### ISSUE TITLE
Error when iterating FastqReader(fastq.gz)

### ISSUE THREAD

**Issue by Puriney** [simon-anders/htseq#32]
```  
File "foo_iter_fastqgz.py", line 9, in <module>
    for read in FastqReader(fpath):
  File "/ifs/home/yy1533/SHARED_APP/miniconda3/lib/python3.6/site-packages/HTSeq/__init__.py", line 392, in __iter__
    if not qual.endswith( "\n" ):
TypeError: endswith first arg must be bytes or a tuple of bytes, not str
```

There was no problem when running FastqReader(fastq) but error was reported when using fastq.gz file.

Py3.6 with HTseq installed by `pip install HTSeq`.

**Comment by iosonofabio** [comment]
Thx that's a bug will fix ASAP

**Comment by iosonofabio** [comment]
fixed in 7fae5ea, will be in 0.9.0
